### PR TITLE
App control PR

### PR DESCRIPTION
**Pull request by Javed-Shahbaz**
## In this project, 
- Create all controllers and views for your Blog app. 
- Add simple placeholders for your views with plain HTML.
- Use controllers to handle requests and render empty views.
- Understand Rails naming conventions.
- Use params from browser requests in a safe way.

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+class UsersController < ApplicationController
+end
```
Comment (Zuhaib042) [inline app/controllers/users_controller.rb]: - [ ] There are no action (methods) in this controller. kindly consider adding both `index` and `show` action(method) here. even empty, these action(methods) are necessary for the view to display properly. example is below.
```
def index
end
```

File: app/controllers/posts_controller.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+class PostsController < ApplicationController
+end
```
Comment (Zuhaib042) [inline app/controllers/posts_controller.rb]: - [ ] There are no action (methods) in this controller. kindly consider adding `index` and `show` action(method) here. even empty, these action(methods) are necessary for the view to display properly. example is below.
```
def index
end
```

File: app/views/users/index.html.erb [hunk 3]
```diff
@@ -0,0 +1 @@
+<h1>This page shows all users</h1>
```
Comment (Zuhaib042) [inline app/views/users/index.html.erb:1]: - [ ] Just as you added `app/views/users/index.html.erb` and `app/views/users/show.html.erb`.
Also, consider adding `app/views/posts/index.html.erb` and `app/views/posts/index.html.erb` files with relevant HTML inside as you added here.
requirement:
For each URL you should create:

A route.
An action in the correct controller.
A view file.

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @Javed-Shahbaz 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✅ Added controllers in the app
✅ Successfully added routes for those controllers 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Munsa1 — APPROVED** [review]
Hi @Javed-Shahbaz, 
Approved 1

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Controllers Have Corresponding Views :heavy_check_mark: 
Made Requested Changes :heavy_check_mark: 
Only Necessary Endpoints Have Been Exposed :heavy_check_mark: 

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**